compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -673,6 +673,15 @@ OMR::Z::CodeGenerator::CodeGenerator()
       _callsForPreloadList = new (self()->trHeapMemory()) TR::list<TR_BranchPreloadCallData*>(getTypedAllocator<TR_BranchPreloadCallData*>(comp->allocator()));
       }
 
+<<<<<<< HEAD
+=======
```

Comment:
@dhong44 looks like some changes needed here.

Reply:
Sorry, I must have missed this when fixing the merge conflicts.